src/services/user_api/models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    min: 3,
+    max: 255
+  },
+  email: {
+    type: String,
+    required: true,
+    min: 6,
+    max: 255
+  },
+  password: {
+    type: String,
+    required: true,
+    min: 6,
+    max: 1024
+  },
+  date: {
+    type: Date,
+    default: Date.now
+  }
+});
+
+module.exports = mongoose.model('User', userSchema);
```

Comment:
migrate to export by default, also take time to know when to use **module.exports, export by default and export**